mindsdb/integrations/handlers/stabilityai_handler/stabilityai.py

```diff
@@ -0,0 +1,79 @@
+from stability_sdk import client
+from stability_sdk.client import generation
+from PIL import Image
+import io, requests
+
+class StabilityAPIClient:
+    
+    def __init__(self, api_key, dir_to_save, 
+                 engine="stable-diffusion-xl-1024-v1-0", upscale_engine="esrgan-v1-x2plus"):
+        self.api_key = api_key
```

Comment:
Please mention in the comments about what this function does.

Reply:
Done